src/DoctrineGui/Controller/DoctrineGuiController.php

```diff
@@ -361,8 +366,6 @@ public function clientManageAction()
             $clientObject->addScope($scopeObject);
         }
 
-        $clientObject->setUser();  //@TODO Add user
-
         if ($new_password != '')
```

Comment:
Use camel case for non-foreign key variables.  Should be $newPassword.  Also no need to check for empty string, use `if ($newPassword) {`